### PR TITLE
Visual improvements to subscription table

### PR DESCRIPTION
**Pull request by ClementChaumel**
## Done

- Fix /advantage 's table alignment according to this [design](https://app.zeplin.io/project/601a8f0a371bc62234c9fc39/screen/601bbaaafb4831787d52354b)

## QA

- go to https://ubuntu-com-9220.demos.haus/advantage?test_backend=true
- login
- check that the table looks as specified

## Issue / Card

Fixes https://github.com/canonical-web-and-design/master-epics/issues/245

## Screenshots

before 
image

after

image

Small screen
image

### CODE REVIEW

File: templates/advantage/index.html [hunk 1]
```diff
@@ -225,10 +225,10 @@ <h4>Ubuntu 16.04 and 18.04 LTS</h4>
                 <tr class="p-table__row">
                   <th class="u-no-padding u-vertically-center"><span style="padding-left: 0.5rem;">{{ account }}</span></th>
                   <th>UA plan</th>
-                  <th class="u-align--center">Attached</th>
-                  <th class="u-align--center">ESM</th>
+                  <th class="">Attached</th>
+                  <th class="">ESM</th>
```
Comment (anthonydillon) [inline templates/advantage/index.html:229]: Remove empty classes

File: templates/advantage/index.html [hunk 2]
```diff
@@ -248,17 +248,17 @@ <h4>Ubuntu 16.04 and 18.04 LTS</h4>
...
+                      <td class="{% if new_subscription_id == contract['contractInfo']['id'] %} p-table--open{% endif %}">
                         <span class="u-hide">{{ contract }}</span>
-                        <button class="u-toggle" aria-controls="#expanded-row-token-{{ outer_loop.index }}-{{ loop.index }}" aria-expanded="{% if new_subscription_id == contract['contractInfo']['id'] %} p-table--open{% endif %}" data-shown-text="Hide" data-hidden-text="Show">
+                        <button class="u-no-padding--left u-toggle" aria-controls="#expanded-row-token-{{ outer_loop.index }}-{{ loop.index }}" aria-expanded="{% if new_subscription_id == contract['contractInfo']['id'] %} p-table--open{% endif %}" data-shown-text="Hide" data-hidden-text="Show">
                           {{ contract['machineCount'] }}&nbsp;<i class="p-icon--contextual-menu {% if default_open_cell %}u-rotate{% endif %}">Open</i>
                         </button>
                       </td>
 
-                      <td class="u-align--center">
+                      <td class="">
```
Comment (anthonydillon) [inline templates/advantage/index.html:258]: empty class

**Review by anthonydillon — APPROVED** [review]
LGTM :+1:

**Comment by anasereijo** [comment]
@wgx as I mentioned this morning I only realised we have <new> tags when I saw this. Is this something we want to keep? If so, I might need to rethink the layout of the table as it looks a bit weird now. Any thoughts?

**Comment by wgx** [comment]
@anasereijo I think we should keep it for now - it's not great, but let's fix in another pass.

**Comment by anasereijo** [comment]
@ClementChaumel as discussed here is a list of things to amend:

- [ ] remove vertical border from dropdowns
- [ ] chevrons aligned and next to the longest title (8px between text and title)
- [ ] fix alignment on table labels

Thanks!

**Comment by ClementChaumel** [comment]
@anasereijo I made the changes what do you think? 
image
---
image

**Pull request merged**